cube/android/README.md

```diff
@@ -22,7 +22,7 @@ Copy contents of parsed_header into generate_vulkan_wrapper.py between the secti
 Then run the script:
 
     popd
-    ./generate_vulkan_wrapper.py ../common/vulkan_wrapper.h
-    ./generate_vulkan_wrapper.py ../common/vulkan_wrapper.cpp
+    ./generate_vulkan_wrapper.py ../cube/android/vulkan_wrapper.h
+    ./generate_vulkan_wrapper.py ../cube/android/vulkan_wrapper.cpp
```

Comment:
So is android the only thing that uses vulkan_wrapper? I was half under the impression that it was used for something else (possibly not in the repo?)

Reply:
As far as I can tell these files are only used for android for vkcube. I don't see anything indicating otherwise.